doc/man7/provider-cipher.pod

```diff
@@ -306,7 +306,7 @@ beginning of the output buffer, do "in place" encryption of the payload and
 write that to the output buffer, and finally add the tag onto the end of the
 output buffer.
 
-Whether encrypting or decrypting the value written to B<*outl> in the
+Whether encrypting or decrypting the value written to I<*outl> in the
```

Comment:
Is this still a parameter name with the leading `*`?
Would `*I<outl>` be better?

Reply:
Leave the pointer bolded please.

Reply:
Bolded?  You mean italic, I assume.  I'm torn...  is there a standard library example we can look at?

Although, I would go for `B<*>I<name>`...

Reply:
`strtol` perhaps?

Reply:
Thanks.  Perfect example.  [strtol(3)](https://linux.die.net/man/3/strtol) shows a similar example, and the `*` are in italics (depending on your rendering, you have to examine the HTML to see it)

In other words, I leave as is.

Reply:
Okay, it came out underlined here like all the other highlights.  I've no idea where MacOS stores the raw `man` pages (which aren't HTML) or I'd have checked there.